lib/Doctrine/Common/Cache/MemcachedCache.php

```diff
@@ -92,6 +92,16 @@ protected function doSaveMultiple(array $keysAndValues, $lifetime = 0)
     /**
      * {@inheritdoc}
      */
+    protected function doDeleteMultiple(array $keys)
+    {
+        $stat = $this->memcached->deleteMulti($keys);
+        return count(array_unique($stat) > 1)
+            || end($stat) === true;
```

Comment:
Needs testing against:

 * empty set of keys
 * one key (success)
 * one key (failure)
 * multiple keys (success)
 * multiple keys (a failure)
 * multiple keys (all failures)